engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -537,18 +537,18 @@ public float getBlockLightIntensityAt(Vector3f position) {
 
     @Override
     public String getMetrics() {
-        String stringToReturn = "";
-        stringToReturn += renderableWorld.getMetrics();
-        stringToReturn += "Empty Mesh Chunks: ";
-        stringToReturn += statChunkMeshEmpty;
-        stringToReturn += "\n";
-        stringToReturn += "Unready Chunks: ";
-        stringToReturn += statChunkNotReady;
-        stringToReturn += "\n";
-        stringToReturn += "Rendered Triangles: ";
-        stringToReturn += statRenderedTriangles;
-        stringToReturn += "\n";
-        return stringToReturn;
+        StringBuilder stringToReturn = new StringBuilder("");
```

Comment:
I'm not going to fight over this but this is a good reference:

http://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java